src/Client.php

```diff
@@ -373,6 +373,14 @@ public function send($method, $payload, $path, $params = [], array $headers = []
             throw new Http\Exception\UnprocessableEntityException($content);
         }
 
+        if ($response->getStatusCode() === 429) {
+            throw new Http\Exception\TooManyRequestsException(
+                $response->getHeaderLine('X-Rate-Limit-Reset'),
```

Comment:
First argument in `TooManyRequestsException` expected `Retry-After`. The `X-Rate-Limit-Limit` header has another meaning.